src/sentry/api/event_search.py

```diff
@@ -396,9 +401,15 @@ def visit_numeric_filter(self, node, children):
             )
             return self._handle_basic_filter(search_key, "=", search_value)
 
-    def visit_aggregate_filter(self, node, children):
-        (search_key, _, operator, search_value) = children
+    def handle_negation(self, negation, operator):
         operator = operator[0] if not isinstance(operator, Node) else "="
+        if self.is_negated(negation):
+            return NEGATION_MAP.get(operator, "=")
```

Comment:
Should the default be `!=` here as you're looking for a negation?

Reply:
oh lol, yeah that makes way more sense :+1: